### DISCUSSION TITLE
Attribute "DOWNLOAD" is not working

### DISCUSSION THREAD

**Discussion by crismo35** [atom/atom#22695] [Q&A]
I am starting in this thing of WEB programmer, and when I tried to add the attribute download to ATOM
"< a href="../IMG/1.jpg" download="../IMG/1.jpg">Descarga</a 
![problema ATOM DOWNLOAD](https://user-images.githubusercontent.com/86990962/124526133-d10a1600-ddcf-11eb-8428-17f8b56ceaa0.PNG)
>"
When I test the link, the link don't work and it shows in the web instead download it, please help me with this issue, thank you

**Comment by idleberg** [comment]
The download attribute is meant to contain a file name that is used on download (e.g. "MyCoolImage.jpg"). It makes no sense to put in path, let alone the same that is already used as href.

Anyway, this forum is about Atom, not for basic HTML support.
